### ISSUE TITLE
Improve CQ Statements by :Name aliases

### ISSUE THREAD

**Issue by NielsKorschinsky** [IBM/spectrum-protect-sppmon#39]
Copied issue from old internal github.
See https://github.com/influxdata/influxdb/issues/7332

The Idea is to use a influx-keyword to retain the original field name if you use a aggregate function like `mean(*)`. Currently it appends a prefix everywhere, the only way to keep the old name is to manually place an alias behind each query: `mean(cpu) as cpu`.
This workarround is currently in place, but requires a lot of lines of code and makes the code way harder to maintenance. Once the Issue of influxdb is resolved, it should be quite easy to look into this again.
Until then: Blocked.

The requirement for the same naming are the different retention policies. To have the 90 day dashboard switch between 90 days and INF, it is required to have exactly the same field names. Otherwise another dashboard is required, which makes the Grafana dashboards even harder to update.

**Comment by NielsKorschinsky** [comment]
I've noticed that not every calculation is an exact rename. On some downsamples I've named them if using a sum function with the prefix `sum_`. This could create some issue when using the 90-days dashboard on INF scale. 
This should *not* be an issue, if starting on the 14 day scale, since the 90 day dashboard only affects the 90/inf-RP where it would be the same.

If we start experience issues there I will have to remove the prefix from all downsamples 90 days to INF, where those names arn't exactly the same.
